src/index.ts

```diff
@@ -76,29 +76,33 @@ class VueBreadcrumbs implements PluginObject<ComponentOptions<Vue>> {
             },
             this.$breadcrumbs.map((crumb: RouteRecord, index: number) => {
               if (crumb?.meta?.breadcrumb) {
-                return createElement(
-                  'li',
-                  {
-                    class: {
-                      'breadcrumb-item': true
-                    },
-                    props: {
-                      key: index
-                    }
-                  },
-                  [
-                    createElement(
-                      'router-link',
-                      {
-                        props: {
-                          to: { path: this.getPath(crumb) },
-                          tag: index !== this.$breadcrumbs.length - 1 ? 'a' : 'span'
-                        }
+                let label = this.getBreadcrumb(crumb.meta.breadcrumb);
+                if (label !== '') {
+                  
+                  return createElement(
+                    'li',
+                    {
+                      class: {
+                        'breadcrumb-item': true
                       },
-                      ` ${this.getBreadcrumb(crumb.meta.breadcrumb)}`
-                    )
-                  ]
-                )
+                      props: {
+                        key: index
+                      }
+                    },
+                    [
+                      createElement(
+                        'router-link',
+                        {
+                          props: {
+                            to: { path: this.getPath(crumb) },
+                            tag: index !== this.$breadcrumbs.length - 1 ? 'a' : 'span'
+                          }
+                        },
+                        ` ${label}`
```

Comment:
Would need to change here too

Reply:
> Would need to change here too

oh yes you're right i missed that moment